### DISCUSSION TITLE
坐等验证后的工程文件 除了2块有问题的板子 其他板子有做改动吗  没有的话我就直接打板开团啦

### DISCUSSION THREAD

**Discussion by Ericbai0808** [peng-zhihui/HelloWord-Keyboard#46] [Q&A] [answered]
坐等验证后的工程文件 除了2块有问题的板子 其他板子有做改动吗  没有的话我就直接打板开团啦

**Comment by peng-zhihui** [comment, marked as answer]
建议等等

**Comment by xdeng** [comment]
你的团是什么样的团可以介绍下吗？合适的话想上车。

**Comment by Ericbai0808** [comment]
瀚文PCB+SMT发车群 
897476469

**Comment by peng-zhihui** [comment]
我这周已经在嘉立创重新打样并且SMT了，而且把几乎所有元器件都替换为可以直接在立创下单的，省得大家单独去找零件，下周六更新工程。

**Reply by tklovejk** [reply]
麻烦问下，是换了很多元件吗，还是可以平替的，因为好多人已经提前把元件买好只等pcb了。谢谢大佬！

**Comment by peng-zhihui** [comment]
元件没替换，只是把元件封装换成了可以直接下单的
